### DISCUSSION TITLE
Guideline: Atomic Coding

### DISCUSSION THREAD

**Discussion by waldo1001** [microsoft/alguidelines#65] [BC Best Practices]
(also find the right name for it)

**Comment by StefanMaron** [comment]
I think this could go along with code complexity and maintainability index.

https://en.wikipedia.org/wiki/Cyclomatic_complexity
https://docs.microsoft.com/en-us/visualstudio/code-quality/code-metrics-maintainability-index-range-and-meaning?view=vs-2022

I included a warning in the BC.LinterCop to warn on procedures and tirggers if they reach a certiain complexity
https://github.com/StefanMaron/BusinessCentral.LinterCop#rules

See Rule LC0010
